### PR TITLE
Added Maruti specific fonts

### PR DESCRIPTION
**Pull request by sneharora1**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--maruti-playground--buuhuu.hlx.live/
- After: https://fonts--maruti-playground--buuhuu.hlx.live/

### CODE REVIEW

File: styles/fonts.css [hunk 1]
```diff
@@ -16,3 +16,83 @@
...
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2122, U+2191, U+2193, U+2212, U+2215, U+FEFF, U+FFFD;
 }
+
+/* latin-ext */
+@font-face {
+  font-family: Lato;
+  font-style: normal;
+  font-weight: 100;
+  font-display: swap;
+  src: url("https://fonts.gstatic.com/s/lato/v24/S6u8w4BMUTPHh30AUi-qNiXg7eU0.woff2") format('woff2');
```
Comment (anagarwa) [inline styles/fonts.css:26]: @sneharora1 this should be referenced the local file.

Comment (sneharora1) [reply styles/fonts.css]: @anagarwa  Seems like my changes werent saved :(
Fixed it now.

**Comment by anagarwa** [comment]
@sneharora1  IMO we should update global variable as well https://github.com/buuhuu/maruti-playground/blob/main/styles/styles.css#L23 here

**Comment by anagarwa** [comment]
@sneharora1 IMO we might not this change at all , Since MSIL has already done this in their repos

**Pull request merged**